srcpkgs/glm/template

```diff
@@ -30,6 +32,5 @@ post_install() {
 		dos2unix "$f"
 	done
 	rm -f "${DESTDIR}/usr/include/CMakeLists.txt"
```

Comment:
I think they meant
```suggestion
	rm -f "${DESTDIR}/usr/include/glm/CMakeLists.txt"
```

Reply:
Yeah that seems to be the path that was intended. I've added this change to the PR